frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -405,6 +405,7 @@ describe('Kubernetes resource CRUD operations', () => {
             '/k8s/ns/openshift-monitoring/monitoring.coreos.com~v1~Alertmanager/main',
             '/settings/cluster',
             '/monitoring/query-browser',
+            '/search/all-namespaces?kind=capabilities.3scale.net~v1alpha1~API',
```

Comment:
We should pick a kind that exists by default on the cluster without needing an operator. I'd also add a comment explaining.

Reply:
Maybe

```suggestion
            // Test loading search page for a kind with no static model.
            '/search/all-namespaces?kind=config.openshift.io~v1~Console',
```